src/Admin/AbstractAdmin.php

```diff
@@ -799,6 +799,15 @@ public function preBatchAction($actionName, ProxyQueryInterface $query, array &$
     public function getFilterParameters()
     {
         $parameters = [];
+        if ($this->getModelManager() && \is_array($this->getModelManager()->getDefaultSortValues($this->getClass()))) {
```

Comment:
The check is not necessary.

I think that getModelManager always return a ModelManager.
And the type hint of the return value of getDefaultSortValues is `array`.

This allow to write : 
```
$parameters = array_merge(
            $this->getModelManager()->getDefaultSortValues($this->getClass()),
            $this->datagridValues, // NEXT_MAJOR: Remove this line.
            $this->getDefaultSortValues(),
            $this->getDefaultFilterValues()
        );
```